workflow/kfdrc-somatic-variant-workflow.cwl

```diff
@@ -532,7 +532,7 @@ steps:
     out: [bam_file]
   prepare_regions_padded:
     run: ../sub_workflows/prepare_regions.cwl
-    when: $(inputs.wgs_or_wxs == 'WXS')
+    when: $(inputs.wgs_or_wxs == 'WXS' && inputs.run_tool.some(function(e) { return e }))
```

Comment:
Oh interesting, so as long as at least one of the values of `[runtime_validator/run_vardict, runtime_validator/run_lancet, runtime_validator/run_controlfreec, runtime_validator/run_cnvkit]` is true, then it's considered a go is the interpretation?